openapi/platform/platform.yaml

```diff
@@ -0,0 +1,507 @@
+                      The complete phone number for the location.
+                      Formating characters may be included. 
+
+                      **Extentions**
+
+                      It is recommended that extentions are included at the end of the number after an appropriate seperator: Pause: `,` Wait: `;` Default: `ext.`
+
+                      You may test the parsibility of a number using https://phonenumbers.temba.io/
+                required:
+                  - raw
+            serviceAreaBusiness:
+              type: boolean
+              description: When true the address will be used as the center of the area that this location services instead of being displayed.
+        relationships:
+          type: object
+          properties:
+            owner:
+              type: object
+              description: A link to the organization that owns this record. Account Locations are typically owned by a partner.
+              properties:
+                data:
+                  type: object
+                  required:
+                    - type
+                    - id
+                  properties:
+                    type:
+                      type: string
+                      default: organizations
+                      enum:
+                        - organizations
+                    id:
+                      type: string
+                      example: ABC
+              required:
+                - data
+    accountContacts:
+      title: Account Contact
+      type: object
+      description: |-
+        An account contact is a person who works for the account that your sales, fulfillment, billing and support teams may send communications to reguarding the account.
+
+        At least one of `name.first`, `name.last`, `phone.raw` or `email` must be provided when creating a contact.
+      properties:
+        type:
+          type: string
+          default: accountContacts
+          example: accountContacts
+          enum:
+            - accountContacts
+        id:
+          type: string
+          description: The id will be assigned by the server and must be included on all update requests. Values sent during creation will be ignored.
+        attributes:
+          type: object
+          properties:
+            name:
+              type: object
+              properties:
+                first:
+                  type: string
+                  maxLength: 25
+                  description: The given name of the contact
+                last:
+                  type: string
+                  maxLength: 25
+                  description: The surname of the contact
+                greeting:
+                  type: string
+                  maxLength: 25
+                  description: The name to use in email messages. Defaults to first name.
+            phone:
+              type: object
+              description: The primary phone number for the contact
+              properties:
```

Comment:
The intention is that additional readonly properties will be added in the future with the phone number formatted for specific use cases. (Local, international, automated dialing, etc)